epub33/rs/index.html

```diff
@@ -965,6 +966,15 @@ <h4>The <code>rendition:layout</code> Property</h4>
 								rendering.</p>
 						</dd>
 					</dl>
+
+					<p>When one of the <a href="https://www.w3.org/TR/epub-33/#layout-overrides"
+								><code>rendition:layout</code> overrides</a> [[EPUB-33]] is specified in the
+							<code>properties</code> attribute on a spine <code>itemref</code> element, Reading Systems
+						MUST apply the same rendering logic to the referenced resource.</p>
+
+					<p>If more than one <code>rendition:layout</code> override is specified in the
```

Comment:
> When one of the `rendition:layout` overrides is specified in the `properties` attribute on a spine `itemref` element, Reading Systems MUST apply the same rendering logic to the referenced resource.

Same rendering logic as the override? Maybe:

> When a value of the `properties` attribute of the `itemref` element overrides the global value of `rendition:layout`, Reading Systems MUST use the rendering logic specified on that spine item.

Reply:
> When a value of the properties attribute of the itemref element overrides the global value of rendition:layout

I got bogged down trying wording like this because you then have to define how to parse the attribute. The attribute has a value that is the whitespace separated list of tokens. We might get away with "contains", but I still think we need to reference the spine overrides otherwise there's no context in this document.

> Reading Systems MUST use the rendering logic specified on that spine item.

The paragraph was intended to follow on from the rendering logic for each global value in the list that immediately precedes the paragraph (i.e., use the applicable logic above). We're still missing that connection.

This might still need some more tweaking, but:

> When the value of the `properties` attribute on an `itemref` element contains an [override of the global rendition:layout property](http://www.w3.org/TR/epub-33/#) [[EPUB-33]], Reading Systems MUST use the rendering logic associated with the override's global value on that spine item (e.g., a spine item that specifies `layout-prepaginated` is rendered following the requirements of the global `pre-paginated` value).

Reply:
I've just committed a fairly substantive change for this. It's annoyingly duplicative to have the same prose for every rendition property, so I've created a new section under the spine processing for handling overrides. I was going to put it in the FXL properties section, but it also applies to the flow property, so keeping it with the spine seemed most appropriate.

I also noticed that the orientation property didn't expand on its values. It just had a generic statement saying to use the orientation specified, but that didn't account for the auto value. I've added a dl list to define these like the other definitions.